### PR TITLE
Review of installation, update and uninstallation procedures

### PR DESCRIPTION
**Pull request by abravosuse**
### PR creator: Description

Revision on installation, update and uninstallation procedures.

### CODE REVIEW

**Review by tomschr — CHANGES_REQUESTED** [review]
@abravosuse Thanks for the addition. Much appreciated. :+1: I have some suggestions, hope they make sense.

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -319,25 +318,34 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
+       on a single &k8s; cluster node. You can use the Trento helm chart in order to install Trento Server
+       on a existing &k8s; cluster as follows:
       </para>
       <procedure>
         <step>
           <para>Install Helm:</para>
           <screen>curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash</screen>
+        </step>
+         <step>
+          <para>Connect Helm to the existing &k8s; cluster.</para>
```
[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:334]: We cannot be more specific here, it really depends on the customer setup.

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -351,15 +359,16 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
+       All you need is a small server or VM (see <xref
           linkend="sec-trento-server-requirements" /> for minimum requirements)
         and follow steps in <xref linkend="pro-trento-manually-installing" /> to
         get &t.server; up and running. </para>
      
      <important>
       <title>Deploying Trento on K3s</title>
       <para>
-        &suse; does not recommend using Trento on K3s for production purposes.
+        The following procedure deploys &t.server; on a single-node K3s cluster. Such set up is not recommended for production purposes.
```
[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:371]: The recommended case is to have a person with K8s knowledge and have him/her deploy -manually- Trento Server components on an existing/new cluster in a production fashion.

File: trento/xml/article_sap_trento.xml [hunk 3]
```diff
@@ -387,25 +396,28 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
-          Note that the experimental flag is not needed as of Helm version 3.8.0.
+          When using a Helm version lower than 3.8.0, a experimental flag must be set before the helm command:
         </para>
+         <screen>HELM_EXPERIMENTAL_OCI=1 helm upgrade \
+   --install trento-server oci://registry.suse.com/trento/trento-server \
+   --set trento-web.adminUser.password=<replaceable>ADMIN_PASSWORD</replaceable></screen>
       </step>
        <step>
-        <para> Monitor the creation and start-up of the Trento &k8s; pods and wait until they are all in running status:</para>
+        <para> Monitor the creation and start-up of the Trento Server pods and wait until they are all in ready/running status:</para>
```
[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:420]: In this case it has to be ready AND running.

File: trento/xml/article_sap_trento.xml [hunk 4]
```diff
@@ -2070,61 +2082,36 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
-        </para>
-        <para>
-          If email alerting is enabled in the environment, then the update command
-          should include the corresponding options:
-        </para>
-        <screen>HELM_EXPERIMENTAL_OCI=1 helm upgrade \
+   <para>A few things to consider:</para>
+      <itemizedlist>
+        <listitem>
+          <para>Remember to add set the helm experimental flag if you are using a version of Helm lower than 3.8.0.
```
[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:2096]: set

File: trento/xml/article_sap_trento.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml:420]: Ok, replaced "or" with "and". It should work now and you can accept it.

File: trento/xml/article_sap_trento.xml [hunk 5]
```diff
@@ -2070,61 +2082,35 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2096]:
```suggestion
          <para>Remember to set the helm experimental flag if you are using a version of Helm lower than 3.8.0.
```

**Pull request merged**